source/extensions/quic_listeners/quiche/platform/quic_mock_log_impl.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include <string>
+
+#include "common/common/assert.h"
+
+#include "extensions/quic_listeners/quiche/platform/quic_logging_impl.h"
+
+#include "gmock/gmock.h"
+
+namespace quic {
+
+class QuicEnvoyMockLog : public QuicLogSink {
```

Comment:
nit: Comment about what this class does.

Reply:
Done.